draft-pauly-dprive-oblivious-doh.md

```diff
@@ -171,19 +176,10 @@ to indicate that this request is an Oblivious DoH query intended for proxying. C
 also SHOULD set this same value for the HTTP Accept header.
 
 A correctly encoded request has the HTTP Content-Type header "application/oblivious-dns-message",
-uses the HTTP POST method, and contains "targethost" and "targetpath" variables.
-The Proxy MUST validate these parameters and construct a URI Template as follows:
-
-1. Let $TARGET be the "targethost" parameter from the Client's request, encoded as the
-concatenation of a "host" value (Section 3.2.2 of {{!RFC3986}}) and, optionally, the
-":" character followed by port (Section 3.2.3 of {{!RFC3986}}). If this parameter cannot
-be encoded as such, the Client's request is incorrectly encoded.
-1. Let $PATH be the "targetpath" parameter from the Client's request, which MUST be a URI
-Template {{!RFC6570}}. If this parameter is not a valid URI Template, the Client's request
-is incorrectly encoded.
-1. The Target request HTTPS URI Template is "https://$TARGET$PATH".
-
-The resulting URI from this Template is used to forward the Client's request to the Target.
+uses the HTTP POST method, and contains "targethost" and "targetpath" variables. Let
+$TARGET be the "targethost" variable and $PATH be the "targetpath" variable from this
+request, encoded as described above. The Proxy constructs a HTTP request to forward the
```

Comment:
Doesn't the proxy need to percent-decode the path first?

Reply:
Good catch -- this should read "decoded" from the request.

Reply:
```suggestion
request. The Proxy constructs a HTTP request to forward the
```